libpod/plugin/volume_api.go

```diff
@@ -153,6 +153,9 @@ func GetVolumePlugin(name string, path string) (*VolumePlugin, error) {
 	// And since we can reuse it, might as well cache it.
 	client := new(http.Client)
 	client.Timeout = defaultTimeout
+	if time.Duration(timeout)*time.Second > defaultTimeout {
```

Comment:
Why can't I specify a timeout < defaultTimeout?  If I can't shouldn't this return an error?

Reply:
I think allowing a sub 5s timeout won't work for any driver, I will return an error and document